package.json

```diff
@@ -12,10 +12,12 @@
     "node": ">=12.16"
   },
   "scripts": {
-    "clean": "rm -rf dist/*",
+    "clean": "rimraf dist",
+    "esbuild:prod": "node ./esbuild/prod.js",
+    "esbuild:dev": "node ./esbuild/dev.js",
     "prebuild": "run-s clean test",
-    "build": "microbundle-crl --no-compress --format modern,cjs --css-modules false",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
+    "build": "run-s clean esbuild:prod",
```

Comment:
`clean` runs twice here because of the rebuild script. Isn't that going to cause trouble?

Reply:
```suggestion
    "build": "esbuild:prod",
```

Reply:
I kept it in just because I am paranoid and wanted to have removal of the dist directory consistently colocated with the build script (prod/dev) :) It is ok to remove it. On the other hand I don't think it does any harm.